### ISSUE TITLE
Feedback to empty Search

### ISSUE THREAD

**Issue by thery** [coq-community/vscoq#36]
As Search goes to the output window it would be nice that there will be delimitators (specially when the result is void)

Search nat bool.

could output

"Search nat bool"  gives : 
====================

mult_n_Sm: forall n m : nat, n * m + n = n * S m

====================

**Comment by pseudohuman92** [comment]
An option to clear the output before Search prints its result would be great too.

**Comment by fakusb** [comment]
For the record: there is a button on the RHS of the "select Output" dropdown menu that clears the search manually, but an automatic option one seems like a good idea

**Comment by thery** [comment]
@fakusb yes as a matter of fact my usual interaction is that  I do a `Search` then I realise that I can't figure out where the output start. So I do a clean and redo the `Search` :-(
